flink-cdc-base/src/main/java/com/ververica/cdc/connectors/base/source/assigner/state/PendingSplitsStateSerializer.java

```diff
@@ -150,7 +151,7 @@ private void serializeSnapshotPendingSplitsState(
         writeRemainingSplits(state.getRemainingSplits(), out);
         writeAssignedSnapshotSplits(state.getAssignedSplits(), out);
         writeFinishedOffsets(state.getSplitFinishedOffsets(), out);
-        out.writeBoolean(state.isAssignerFinished());
+        out.writeInt(state.getSnapshotAssignerStatus().getStatusCode());
```

Comment:
The state seems to be incompatible with the previous version, do we need to upgrade the serializer version?

Reply:
Good point! Addressed in https://github.com/ververica/flink-cdc-connectors/pull/1838/commits/89a190a50c1d1398ae2f20193a8206569454481e